wireapp/wire-android
# Fix duplicated entries in Conversations due to the connect request bug

How to test:
1. Install an APK with the old connection requests bug (eg. commit `5990bf`).
2. Log in as A, send a connection request to B.
3. Log in as B (not necessarily from the same APK: the current master, but also iOS and web are also fine), accept the connection request.
4. Depending on the APKs used you should either 
4a. not see the accepted connection request on the conversations list,
4b. see it, but when clicked, it should open a page without buttons,
4c. see it, but when clicked, it should open another conversation.
5. Install this APK.
6. Just after opening the app, the new conversation should appear/refresh and now it should be usable.
